internal/scripts/deploy-dotcom.ts

```diff
@@ -253,8 +253,14 @@ let didUpdateTlsyncWorker = false
 async function deployTlsyncWorker({ dryRun }: { dryRun: boolean }) {
 	const workerId = `${previewId ?? env.TLDRAW_ENV}-tldraw-multiplayer`
 	if (previewId && !didUpdateTlsyncWorker) {
-		await setWranglerPreviewConfig(worker, { name: workerId })
+		const queueName = `tldraw-multiplayer-queue-${previewId}`
+		await setWranglerPreviewConfig(worker, { name: workerId }, queueName)
 		didUpdateTlsyncWorker = true
+		try {
+			await exec('yarn', ['wrangler', 'queues', 'info', queueName], { pwd: worker })
+		} catch (_e) {
+			await exec('yarn', ['wrangler', 'queues', 'create', queueName], { pwd: worker })
```

Comment:
This is not idempotent, so we first try to list it, then we create it if that fails. Also considered deleting then creating again, but cloudflare doesn't allow it since the queue might have active consumers.